test/examples_test.go

```diff
@@ -189,6 +192,11 @@ func expectedProvenance(t *testing.T, example string, obj objects.TektonObject)
 	return intoto.ProvenanceStatement{}
 }
 
+type UriDigestPair struct {
+	Uri    string
```

Comment:
```suggestion
type URIDigestPair struct {
	URI    string
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms